src/Aspire.Dashboard/Resources/xlf/ControlsStrings.tr.xlf

```diff
@@ -29,7 +29,7 @@
       </trans-unit>
       <trans-unit id="ChartContainerGraphTab">
         <source>Graph</source>
-        <target state="translated">Graf</target>
+        <target state="translated">Graph</target>
```

Comment:
did this go backwards? or is it the same as in English?

Reply:
cc @kvenkatrajan I think you already opened an issue about loc PR's turning things back into English, but I can't find it...